apps/obsidian/src/components/RelationshipSection.tsx

```diff
@@ -423,7 +423,72 @@ const CurrentRelationships = ({ activeFile }: RelationshipSectionProps) => {
     }
 
     setGroupedRelationships(Array.from(tempRelationships.values()));
-  };
+  }, [activeFile, plugin]);
+
+  const deleteRelationship = useCallback(
```

Comment:
I'm seeing a lot of linting errors here.  Are you seeing them?

<img width="1755" height="1097" alt="Image" src="https://github.com/user-attachments/assets/44cdd695-ba0c-4161-a3cd-5a9dbc39f729" />